UXO_Kismet_SF_3.tex

```diff
@@ -0,0 +1,363 @@
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \control{}: Perform engine startup procedure:
+    \begin{checklist}
+        \item Arm the Primary Ignition switch.
+        \item Hold down the Fire button until black smoke is observed.
+        \begin{checklist}[label=$\bullet$]
+            \item In the event of a failed ignition (smoke not observed within 1 minute):
+            \begin{checklist}
+                \item \control{}: Disarm the Primary Ignition switch.
+                \item \control{}: Arm the Secondary Ignition switch.
+                \item \ops{}: Revisit ignition procedure.
+            \end{checklist}
+            \item In the event of a second failed ignition (smoke not observed within 1 minute):
+            \begin{checklist}
+                \item \control{}: Disarm the Secondary Ignition switch.
+                \item \primary{}: Open the Line Vent Valve using the ropes.
+                \item \ops{}: Proceed only when the oxidizer tank has fully vented.
+                \item \primary{} and \secondary{}: Approach the test plumbing.
+                \item \primary{}: Close the Cylinder Valve.
+                \item \control{}: Open the Remote Fill Valve.
+                \item \control{}: Open the Tank Vent Valve.
+                \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+                \item \ops{}: Abort test procedures and proceed to teardown.
+            \end{checklist}
+        \end{checklist}
+        \item \control{}: Start the engine by opening the Injector Valve.
+    \end{checklist}
+    \item \textbf{ALL}: Observe the plume:
+    \begin{checklist}[label=$\bullet$]
+        \item If any unexpected events occur during the engine firing:
+        \begin{checklist}
+            \item \primary{}: Open the Line Vent Valve using the ropes.
+            \item \ops{}: Wait for 3 seconds.
+            \item \control{}: Close the Injector Valve.
+        \end{checklist}
+    \end{checklist}
+    \item \textbf{\textit{PAUSE POINT}}
+    \item \ops{}: Wait for at least 3 minutes before proceeding.
+    \item \daq{}: Confirm that the Oxidizer Tank Pressure Transducer is reading atmospheric pressure.
+    \item \control{}: Open the Tank Vent Valve.
+    \item \primary{} and \secondary{}: Approach the test plumbing.
+    \item \primary{}: Close the Cylinder Valve.
+    \item \control{}: Open the Remote Fill Valve.
+    \item \daq{}: Confirm that the Fill Pressure Transducer is reading atmospheric pressure.
+    \item \primary{}: Disconnect the fill line from the supply cylinder.
+    \item \primary{}: Replace the cap on the nitrous oxide supply cylinder.
+    \item \ops{}: Wait for at least 3 minutes before proceeding.
+    \item \peri{}, \perii{}, and \periii{}: Open the perimeter.
+    \item \daq{}: Continue to monitor thermistor readings and inform \ops{} if the combustion chamber temperature exceeds 190 $^\circ$C.
```

Comment:
Maybe have a point about what you'll do if the temperature gets that high?

Reply:
I'm not sure how well defined that procedure is (and I imagine it'll mostly be played by ear if necessary)